### ISSUE TITLE
Cannot create client when using OFFLINE mode

### ISSUE THREAD

**Issue by junbetterway** [openMF/android-client#1162]
**Summary:** 

Cannot create client when using OFFLINE mode

**Steps to reproduce:** 

1. Toggle the mode into offline
2. Go to Clients and click '+' to add new client
3. Fill up the form then click Submit

**Expected behaviour:** 

Client creation should be successful even if offline mode. 

**Observed behaviour:** 

No error messages. The screen just blinked and then stays on the create-client form page

**Device and Android version:** 

Same behaviour on all devices tested so far. (Android 8.0.0)

**Screenshots:** 

No error message. N/A

**Comment by francisguchie** [comment]
@junbetterway  Yes this is true - i tested it with the latest app from google play

**Comment by therajanmaurya** [comment]
Moved to https://mifosforge.jira.com/browse/MIFOSAC-372
